docker/spack.yaml

```diff
@@ -0,0 +1,20 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs: 
+    - flexpart-ifs@main
```

Comment:
Example of using spack develop for building from a local source (if we want to support building current commit in CICD).
```
spack:
  specs:
  - flexpart-cosmo@main
  config:
    install_tree:
      root: /home/vcherkas/dev/flexpart-cosmo-sandbox/spack-root
  develop:
    flexpart-cosmo:
      path: /home/vcherkas/dev/flexpart-cosmo-sandbox/flexpart
      spec: flexpart-cosmo@=main
```
also here in CICD: https://github.com/C2SM-RCM/flexpart/blob/main/Dockerfile.base#L91